### PR TITLE
Fix for objectives

### PR DESCRIPTION
**Pull request by rushter**
I'm working on the objectives for a backend branch.

Please take a look at my sqrt fix. I don't know why lower bound is epsilon. 

Also, i have two questions:
How can i test both backends in the same time? It seems like it's a problem for a current architecture.
What do you think about my approach to test objectives? I can replace this with static and precomputed examples.

### CODE REVIEW

File: keras/objectives.py [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
-    epsilon = 1.0e-9
-else:
-    epsilon = 1.0e-7
+from keras import backend as K
+from keras.backend.common import _EPSILON as epsilon
 
 
 def mean_squared_error(y_true, y_pred):
-    return T.sqr(y_pred - y_true).mean(axis=-1)
+    return K.mean(K.square(y_pred - y_true))
```
Comment (fchollet) [inline keras/objectives.py]: All these are missing the summing axis.

Comment (rushter) [inline keras/objectives.py]: Yeah. I removed this because -1 will not work on the tensorflow backend.
Should i use 0 instead?

Comment (fchollet) [inline keras/objectives.py]: This is fixed. Please do not submit PRs that are incorrect or unfinished, post a Github issue instead.

**Comment by fchollet** [comment]
Please resubmit when ready to merge.

**Comment by rushter** [comment]
@fchollet Can you answer my questions, please?
